src/macaron/__main__.py

```diff
@@ -29,6 +27,18 @@
 
 def analyze_slsa_levels_single(analyzer_single_args: argparse.Namespace) -> None:
     """Run the SLSA checks against a single target repository."""
+    if not (analyzer_single_args.repo_path or analyzer_single_args.package_url or analyzer_single_args.config_path):
+        # We don't mention --config-path as a possible option in this log message as it going to be move soon.
+        # See: https://github.com/oracle/macaron/issues/417
+        logger.error("Analysis target missing. Please provide package url and/or repo path.")
```

Comment:
```suggestion
      logger.error("Analysis target missing. Please provide a package url (PURL) and/or repository path.")
```

Also, it would be good to add an example PURL and repo path, with a reference to https://github.com/package-url/purl-spec for more examples.

Reply:
Fixed in https://github.com/oracle/macaron/pull/401/commits/f66b0be2bf3c2556eed5885315d39cd66b792306